src/components/Button/index.jsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import styles from "./styles/styles.module.scss";
+
+const Button = ({ text, path, type }) => {
+  let buttonStyle = styles.primary;
+
+  if (type === "success") {
+    buttonStyle = styles.success;
+  }
+
+  return (
+    <Link to={path}>
+      <button className={[buttonStyle, styles.button].join(" ")}>{text}</button>
```

Comment:
Which one is more readable and efficient?
```js
[buttonStyle, styles.button].join(" ")
```
or 
```js
`${buttonStyle} ${styles.button}`
```

Reply:
you are right thankssssss